### DISCUSSION TITLE
syntax to include my own doc content tags in djot files?

### DISCUSSION THREAD

**Discussion by uvtc** [jgm/djot#324] [General]
If I've got a big pile of djot docs, it would be helpful if I could include a line near the top of each one containing a bunch of tags, something like:

~~~
#recipe #tomato #basil #fried
~~~

where some other file might contain

~~~
#recipe #tomato #mozzarella #baked
~~~

The idea is that some simple doc management system could then index all my docs by tag to make searching easier. ("show me all docs containing the tag '#tomato' ")

Is there currently a syntax in djot to include those sorts of tags in a doc?

**Comment by vassudanagunta** [comment]
typically tags are a part of front matter, at the top of the doc. e.g.:

```
---
title: Tagging in djot
author: jgm
tags: ["tagging", "front matter"]
---
```

**Reply by uvtc** [reply]
Thanks. I found the relevant issue #35 regarding djot metadata.
